src/Common/Core/Form.php

```diff
@@ -243,7 +243,7 @@ public function addDropdown(
     ): SpoonFormDropdown {
         $name = (string) $name;
         $multipleSelection = (bool) $multipleSelection;
-        $class = (string) ($class ?? 'form-control fork-form-select');
```

Comment:
Was there styling on this class that needs to be removed?

Reply:
Nope, `form-control` was from bootstrap and `fork-form-select` is an old class that was used long ago in fork